src/ansi-c/c_typecheck_base.cpp

```diff
@@ -262,6 +262,41 @@ void c_typecheck_baset::typecheck_redefinition_type(
   }
 }
 
+static bool is_instantiation_of_flexbile_array(
```

Comment:
This has a typo in the definition of the function (`flexbile`).

Can this get fixed before any merge of the PR?

(The typo is going to make the function less discoverable, me thinks)

Reply:
Thank you for such careful proof reading! This is now fixed.